.gitignore

```diff
@@ -64,6 +64,7 @@ packaging/msi/obj
 simfdb
 tests/oldBinaries
 trace.*.xml
+trace.*.json
```

Comment:
intentional add?

Reply:
Yeah otherwise this shows up when doing git add, etc.. which is annoying